task-manager-backend/src/task/controllers/update-task/update-task.js

```diff
@@ -0,0 +1,30 @@
+const ApiResponse = require("../../../utils/api-response");
+const ApiError = require("../../../utils/api-error");
+const { updateTaskUsecase } = require("../../usecases");
+
+const { TASK_UPDATE_SUCCESS } = require("../../../commons/constants");
+
+module.exports = {
+    updateTask: {
+        path: "/:id",
```

Comment:
same here as above. Give names to routePath, methods and use-cases accordingly as per the comment above

Reply:
same here I have added parent path to app